util/missingcrypto.txt

```diff
@@ -1128,6 +1128,7 @@ TS_TST_INFO_set_time
 TS_TST_INFO_set_tsa
 TS_TST_INFO_set_version
 TS_VERIFY_CTS_set_certs
+TS_VERIFY_CTX_set_certs
```

Comment:
Errr....no. We should *never* be *adding* to the missingcrypto.txt files (this one or the one below). The correct fix is to document the newly added function.

Reply:
What is the purpose of this `missingcrypto.txt` file?

Reply:
make doc-nits complains if you introduce new functions that aren't documented. The idea is that we never add a new undocumented function again. At the point that we introduced that capability in doc-nits there was (obviously) a lot of functions that were at that point still undocumented. So missingcrypto.txt is a list of all of those functions so that doc-nits doesn't complain about them.

We introduced this capability after development on 3.0 had already started, and it highlighted the fact that even in the space of time between 1.1.1 being released and the current state of work on 3.0 we had already added new undocumented functions (see issue #9095).  The missingcrypto111.txt file lists the undocumented functions as at the release of 1.1.1. Ideally we should document all functions that are listed in missingcrypto.txt that aren't in missingcrypto111.txt before 3.0 gets released, i.e. so that we can say that 3.0 has not added any new undocumented functions.

Reply:
Hmmm, the newly added function `TS_VERIFY_CTX_set_certs` is not really a new function since the 'typo' one already exists (the `TS_VERIFY_CTS_set_certs`). So if the `TS_VERIFY_CTX_set_certs` should be documented, then does this mean all the TS_VERIFY prefixed functions should all be documented at one time?

Reply:
> So if the TS_VERIFY_CTX_set_certs should be documented, then does this mean all the TS_VERIFY prefixed functions should all be documented at one time?

That would be nice if someone wants to do it (and might not be that difficult if already documenting this new one). But not strictly necessary. It would be enough for this PR to just document that one function. I really don't think we should be making an exception in this case.

Reply:
Okay, makes sense

Reply:
It's very difficult to document the function(s) for me because I don't know what all the functions exactly do. The documentation should be done by someone who really knows the implementation details of the functions.